### DISCUSSION TITLE
Release 3.2

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#1526] [General]
## What's new, what's changed... 👀 
* Add Raspberry Pi onboard leds control to PC Miner to signal share states by @revoxhere 
* Add username check in config creator to the PC Miner and AVR Miner by @revoxhere 
* Add missing translation strings by @revoxhere 
* Add current config file text in greeting by @revoxhere 
* Add ability to control how often ESP8266 MQTT update messages are sent by @tyrkelko in https://github.com/revoxhere/duino-coin/pull/1496
* Fix PC Miner clearing it's settings by @LDarki 
* Fix DHT readings by @LDarki (Duino IoT)
* Fix mining key check by @revoxhere
* Update printlock in the PC Miner by @revoxhere 
* Update polish translations by @revoxhere 
* Update PC Miner thread notices by @revoxhere 
* Update README_de_DE by @Techcrafter in https://github.com/revoxhere/duino-coin/pull/1499
* Update README_kr_KR by @dnjswndbf in https://github.com/revoxhere/duino-coin/pull/1502
* Update difficulty descriptions by @Techcrafter in https://github.com/revoxhere/duino-coin/pull/1506

## New contributor 👋
* @Techcrafter made their first contribution in https://github.com/revoxhere/duino-coin/pull/1499

### ⚠️ NOTICE: .exe binaries may be detected as a virus by your antivirus software. This is a **false positive** caused by pyinstallers bootloader, read about it [here](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg). Duino-Coin is not a virus.

**Full Changelog**: https://github.com/revoxhere/duino-coin/compare/3.18...3.2
As always, please report any encountered issues. Happy mining!

<hr /><em>This discussion was created from the release <a href='https://github.com/revoxhere/duino-coin/releases/tag/3.2'>Release 3.2</a>.</em>

**Comment by Mifaxav** [comment]
How do I use this tool with mac?

**Reply by Techcrafter** [reply]
Download and install Python on your Mac. Afterwards, download the Duino Code source code .zip, extract it and run the .py file.

**Comment by Mifaxav** [comment]
Alright.. thanks :D

**Comment by Zimpete2** [comment]
I am trying to download this software to my Rasberry Pi Pico using Arduino IDE. I cannot get my Pico to show up under a COM port. any suggestions on how to fix that?

**Reply by Techcrafter** [reply]
Hold the BOOTSEL button on the Pico while pluging it in to the computer.
